x-pack/plugins/security_solution/public/common/mock/mock_cases_plugin.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { mockCasesContext } from './mock_cases_context';
+
+export const mockCases = {
```

Comment:
We already have a mock of the client here `x-pack/plugins/cases/public/mocks.ts` where the security solution can use in their tests (as we do with `core` for example).

Reply:
ah! I knew I was doing this twice. Thanks I'll change it.